### PR TITLE
Read data pipeline

### PR DESCRIPTION
**Pull request by wlangera**
- targets pipeline to read and prepare data
- data validation and pipeline validation
- markdown file to collect and write sampling frames etc.
- "dag van de landbouw" figure
- READMEs for data folders

### CODE REVIEW

**Comment by wlangera** [comment]
@hansvancalster I will send you a mail with more context

File: data/steekproefkaders/.gitignore
Comment (hansvancalster) [inline data/steekproefkaders/.gitignore:1]: Should the csv files in this folder be ignored? Maybe there was a reason to do it (size? already archived on Zenodo?), but otherwise it would be good to have them under version control. I just want to make sure of this is intentional or an oversight.

File: source/markdown/data_controle_2024.Rmd
Comment (hansvancalster) [inline source/markdown/data_controle_2024.Rmd:76]: I think you mean "steekproef" instead of "steekproefkader" in this section?

File: source/targets/data_preparation/_targets.R
Comment (hansvancalster) [inline source/targets/data_preparation/_targets.R:47]: This searches for "qgis_export_sovon_wfs" in "mbag-mas/source/targets/data_preparation/data". I'm not sure why you chose to place the data in that folder instead of "mbag-mas/data". Is there a specific reason?

File: source/targets/data_preparation/_targets/meta/meta
Comment (hansvancalster) [inline source/targets/data_preparation/_targets/meta/meta:1]: This file is only meaningful if different users have access to the same targets data store. There are several options depending on how reproducible we want this to be:

- if all we want is to be able to run the targets pipeline, project members only need access to all the input files. We could share these on gdrive. In that case, if a project member runs `tar::make`, all targets will be recreated and the meta file will be different, creating conflicts. In that case, we better gitignore the meta file
- if on the other hand, we want to be able to fully collaborate on a targets pipeline, we need access to both the input files and the targets data store.  How?
- if none of the above are required, and all that is needed is that someone is responsable to run the pipeline, then only that person needs to have all inputs in the right place and only the pipeline should write relevant outputs that can be used by project members (I think this is the current state of affairs).

This needs some further discussion.

File: source/markdown/data_controle_2024.Rmd
Comment (hansvancalster) [inline source/markdown/data_controle_2024.Rmd:1372]: This check is very important. There are appear to be many problematic cases. How will they be corrected? In the sovon database? We need to be careful how to correct this. I think we discussed this before, but the database is a bit misleading in this regard. When broedcode == 0, the number of **individuals** is counted, whereas in case of broedcode > 0, the agreement is to set this number to 1, but it represents a bird couple: i.e. 2 individuals. This is why so many field workers have change 1 to 2. It is a pity that two different counting units are mixed in the same field. I'm not even sure if we have dealt with this appropriately in our analyses.

File: source/markdown/data_controle_2024.Rmd
Comment (hansvancalster) [inline source/markdown/data_controle_2024.Rmd:1407]:
```suggestion
  - We verwijderen kolommen gerelateerd aan steekproef
```

File: source/markdown/data_controle_2024.Rmd
Comment (hansvancalster) [inline source/markdown/data_controle_2024.Rmd:1535]: Not really necessary for GBIF in my opinion

File: source/markdown/data_controle_2024.Rmd
Comment (hansvancalster) [inline source/markdown/data_controle_2024.Rmd:1539]: Also not necessary

File: source/markdown/data_controle_2024.Rmd
Comment (hansvancalster) [inline source/markdown/data_controle_2024.Rmd:1519]: I noticed "individualCount". See previous comment. It is misleading due to mixing different counting units depending on "behavior". So this should either be explained in metadata or disambiguated.

File: source/markdown/data_controle_2024.Rmd
Comment (hansvancalster) [inline source/markdown/data_controle_2024.Rmd:1563]: Birds seen at larger distance will have more uncertainty. But we do not known how much this is. I would guess about 10% of search distance.

File: source/markdown/data_controle_2024.Rmd
Comment (hansvancalster) [inline source/markdown/data_controle_2024.Rmd:1572]: I would not do that because we have no definition of the complete species pool.

**Review by hansvancalster — COMMENTED** [review]
🚀 This is great work of high quality and quantity 👍

I will wait for approval until we have discussed some of my specific comments.

File: source/targets/data_preparation/_targets.R
Comment (wlangera) [reply source/targets/data_preparation/_targets.R:47]: I will add more information in the files

File: source/targets/data_preparation/_targets.R
Comment (wlangera) [inline source/targets/data_preparation/_targets.R:39]: force download all input data, logisch argument en flexibel zodat je niet alle kaartlagen telkens hebt

File: source/targets/data_preparation/_targets/meta/meta
Comment (wlangera) [reply source/targets/data_preparation/_targets/meta/meta:1]: zal geïgnored worden, data downloaden in aparte functie (later PR)

File: source/markdown/data_controle_2024.Rmd
Comment (wlangera) [reply source/markdown/data_controle_2024.Rmd:1372]: terugkoppelen met Johannes, eens kijken naar jaren in controle bestand

File: source/markdown/data_controle_2024.Rmd
Comment (wlangera) [reply source/markdown/data_controle_2024.Rmd:1519]: 27 (organismQuantity) with individuals (organismQuantityType) broedcode 0
1 (organismQuantity) with breeding pair (organismQuantityType) broedcode > 0

File: source/markdown/data_controle_2024.Rmd
Comment (wlangera) [reply source/markdown/data_controle_2024.Rmd:1563]: min 10 m and further distances 10% of distance 2 plot

File: source/markdown/data_controle_2024.Rmd
Comment (wlangera) [reply source/markdown/data_controle_2024.Rmd:1572]: toevoegen dat het wars geen goed idee is

File: source/markdown/data_controle_2024.Rmd
Comment (wlangera) [reply source/markdown/data_controle_2024.Rmd]: fixed in this commit: [abee402](https://github.com/inbo/mbag-mas/pull/37/commits/abee4024effb3d9f8051d6d1a4bea52cccc72de0)

File: source/targets/data_preparation/_targets/meta/meta
Comment (wlangera) [reply source/targets/data_preparation/_targets/meta/meta:1]: update gitignore: [6811c57](https://github.com/inbo/mbag-mas/pull/37/commits/6811c57e13bd5a40dd0892ebfe3a5643effcd1b2)
remove meta: [1e8e1d5](https://github.com/inbo/mbag-mas/pull/37/commits/1e8e1d56514f483188f2e59b688d7bf6d980bf5e)

The data download will be added to _targets.R once data management review is over, and in a separate PR

File: source/markdown/data_controle_2024.Rmd
Comment (wlangera) [reply source/markdown/data_controle_2024.Rmd]: added comments to file in this commit: [790694b](https://github.com/inbo/mbag-mas/pull/37/commits/790694b551ecd34d0ce7d05c71e55859498a8df3)

File: source/markdown/data_controle_2024.Rmd
Comment (wlangera) [reply source/markdown/data_controle_2024.Rmd:1519]: fixed in: [84d3b34](https://github.com/inbo/mbag-mas/pull/37/commits/84d3b3456c4d266069829fabc21be59514ab1929)

File: source/markdown/data_controle_2024.Rmd
Comment (wlangera) [reply source/markdown/data_controle_2024.Rmd:1372]: done in: [ee880d6](https://github.com/inbo/mbag-mas/pull/37/commits/ee880d69c4c7de32bb4e3b6ead058035ec6bada2)

File: source/markdown/data_controle_2024.Rmd
Comment (wlangera) [reply source/markdown/data_controle_2024.Rmd]: added comments to file in this commit: [790694b](https://github.com/inbo/mbag-mas/pull/37/commits/790694b551ecd34d0ce7d05c71e55859498a8df3)

File: source/targets/data_preparation/_targets.R
Comment (wlangera) [reply source/targets/data_preparation/_targets.R:59]: done in: [fccbf71](https://github.com/inbo/mbag-mas/pull/37/commits/fccbf7166a24fc922a57d8f3ccbde287e73bcf8e)

**Comment by wlangera** [comment]
@hansvancalster I have made adjustments as discussed

**Review by hansvancalster — APPROVED** [review]
Thanks for addressing the review comments. All looks good!

**Pull request merged**
